### DISCUSSION TITLE
TrueNAS, GlueTun, CIDR and Cloudflare Warp - Will it work?

### DISCUSSION THREAD

**Discussion by Ziomal12** [qdm12/gluetun#1491] [Q&A]
Hello,
So I wanted to use Cloudflare Warp as a wireguard VPN for qBittorrent on TrueNAS scale (it uses kubernetes). But it didn't work because cloudflare address (172.16.0.2/32) was included in claster CIDR (172.16.0.0/16), so while configuring IPv4 killswitch there was a conflict.
Can I use GlueTun with those values to achieve working connection or it won't ever work?

**Comment by flleeppyy** [comment]
augh, you're the only person I can find that is also wanting to use cloudflare warp with qbittorrent on truenas scale, and this is over a year old

**Comment by ghost** [comment]
another year passes... :pensive:

**Comment by Ziomal12** [comment]
Now that TrueNAS uses docker it's easily doable. I'll migrate back to TrueNAS in some time (probably over 3 months as I'm busy rn) and report back, although it's easy on docker so I don't suspect any troubles.
